src/DssSpell.sol

```diff
@@ -19,95 +19,62 @@ pragma solidity 0.8.16;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 import { MCD, DssInstance } from "dss-test/MCD.sol";
-import { FlapperInit, FlapperInstance, FlapperUniV2Config } from "src/dependencies/dss-flappers/FlapperInit.sol";
 
-interface DssCronSequencerLike {
-    function addJob(address job) external;
+interface RwaLiquidationOracleLike {
+    function ilks(bytes32 ilk) external view returns (string memory doc, address pip, uint48 tau, uint48 toc);
+    function init(bytes32 ilk, uint256 val, string memory doc, uint48 tau) external;
+    function tell(bytes32 ilk) external;
 }
 
-interface RwaOutputConduitLike {
-    function deny(address usr) external;
-    function hope(address usr) external;
-    function nope(address usr) external;
-    function mate(address usr) external;
-    function hate(address usr) external;
-    function kiss(address who) external;
-    function diss(address who) external;
-    function file(bytes32 what, address data) external;
-    function clap(address _psm) external;
-}
-
-interface RwaUrnLike {
-    function file(bytes32 what, address data) external;
-}
-
-interface ChainlogLike {
-    function removeAddress(bytes32 _key) external;
-}
-
-interface VestLike {
-    function file(bytes32 what, uint256 data) external;
-    function create(address _usr, uint256 _tot, uint256 _bgn, uint256 _tau, uint256 _eta, address _mgr) external returns (uint256 id);
-    function restrict(uint256 _id) external;
-    function yank(uint256 _id) external;
+interface ProxyLike {
+    function exec(address target, bytes calldata args) external payable returns (bytes memory out);
 }
 
 interface GemLike {
     function transfer(address dst, uint256 wad) external returns (bool);
-    function approve(address guy, uint256 wad) external returns (bool);
-    function allowance(address src, address guy) external view returns (uint256);
-}
-
-interface ProxyLike {
-    function exec(address target, bytes calldata args) external payable returns (bytes memory out);
 }
 
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/9cdf05e59054bf3c9f83d3b52717e27775b0267e/governance/votes/Executive%20vote%20-%20July%2014%2C%202023.md' -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/ac220424c65680b0e766061e9a2ad330248a67d6/governance/votes/Executive%20Vote%20-%20August%202%2C%202023.md' -q -O - 2>/dev/null)"
```

Comment:
Here's how I do this:
1. Patrick sends the link (https://github.com/makerdao/community/blob/master/governance/votes/Executive%20Vote%20-%20August%202%2C%202023.md)
2. I open it, go to the 3 dots dropdown and click on "Copy Permalink":
![image](https://github.com/makerdao/spells-mainnet/assets/112016538/5951759a-5a89-4b52-8d8a-5b6163be1957)

It's more straightforward, because I don't have to pick a specific commit myself (even though it's supposed to be the last one, that might not always be true).

Reply:
I'm fine with this.